docs/sources/configuration/integrations/integrations-next/mssql-config.md

```diff
@@ -0,0 +1,72 @@
+---
+aliases:
```

Comment:
I just checked again at the docs and we don't have a doc's page for shimed v2 integrations. So we could delete this.

Reply:
Removed this file.